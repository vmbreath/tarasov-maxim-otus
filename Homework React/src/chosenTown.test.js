import React from 'react';
import renderer from 'react-test-renderer';
import ChosenTown from './ChosenTown';
import {Provider} from 'react-redux'
import configureStore from 'redux-mock-store'

test('ChosenTown renders exactly', () => {
    const initialState = {output: 10}
    const mockStore = configureStore();
    let store = mockStore(initialState);
    const component = renderer.create(
        <Provider store={store}><ChosenTown defaultName={'Москва'}/></Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});