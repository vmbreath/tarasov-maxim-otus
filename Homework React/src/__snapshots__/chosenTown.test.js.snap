// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChosenTown renders exactly 1`] = `
<div
  className="listContainer"
>
  <div
    className="deleteLi"
    onClick={[Function]}
  />
  <div
    className="liElem"
    data-tooltip="Выбрать?"
    onClick={[Function]}
  >
    <h3>
      Москва
    </h3>
  </div>
</div>
`;
